### PR TITLE
Add support and contributing sections

### LINKED ISSUE #13: Add communication channels in `README.md`
Add the various communication channels such as Element, Slack, email etc onto the `README`

### PR DESCRIPTION
**Pull request by Green-Ranger11**
This closes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,14 @@ StanUpMan's UI is built with [svelte](svelte.dev/) and source is hosted at [stan
...
 ## Setting UP
 
-# Contributors
+# Support
+
+Reach out to our maintainer on [element](https://matrix.to/#/#standupman:matrix.org) or contact us at support@standupman.xyz
+
+# Contributing
+
+Feel free to [open an issue](https://github.com/standupman/standupman/issues), create new features, fix bugs and/or improving documentation.
```
Comment (nfebe) [inline README.md]: Leave some space between new sections (51-52) about two lines.

File: README.md [hunk 2]
```diff
@@ -41,7 +41,14 @@ StanUpMan's UI is built with [svelte](svelte.dev/) and source is hosted at [stan
 
 ## Setting UP
 
-# Contributors
+# Support
+
+Reach out to our maintainer on [element](https://matrix.to/#/#standupman:matrix.org) or contact us at support@standupman.xyz
```
Comment (nfebe) [inline README.md]: I would like to change the language a little;

"Join our community chat rooms on element that cover different topics including #bugs, #contributors, #general and more."


Please remove the email part too

File: README.md (hunk 1 above)
Comment (nfebe) [inline README.md]: Some few corrections...

Feel free to "open issues" (not an issue), submit patches to fix bugs and or improve documentation. Please discuss new features with the maintainers in the contributors room.

**Review by nfebe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Green-Ranger11) [reply README.md]: Will do!

Comment (Green-Ranger11) [reply README.md]: Good catch! My bad on that

File: README.md (hunk 2 above)
Comment (Green-Ranger11) [reply README.md]: I have made the change thanks!

**Review by nfebe — APPROVED** [review]

**Comment by nfebe** [comment]
Thanks! Merged.

**Pull request merged**